insteon_mqtt/Modem.py

```diff
@@ -478,7 +486,8 @@ def get_engine_all(self, on_done=None):
 
         # Reload all the device databases.
```

Comment:
I might be worth it to update this comment while we're in here.  Looks like it was copy-pastad from refresh-all.

```suggestion
        # Run Get Engine on all the devices
```